Source/Core/Core/HW/EXI/EXI_DeviceEthernet.h

```diff
@@ -201,6 +203,7 @@ enum class BBADeviceType
 {
   TAP,
   XLINK,
+  BuiltIn,
 #if defined(__APPLE__)
   TAPSERVER,
```

Comment:
Adding `BuiltIn` here means that `TAPSERVER`'s value will be different from what it was in the past.  I think this is fine, though, since `BBADeviceType` is an implementation detail not stored in the configuration (it would be a problem for `EXIDeviceType` if it was there).

It might be better to put `BuiltIn` at the end of the enum to match the order in `EXIDeviceType`, but then it would have a different value depending on whether apple is in use or not.  I don't think that's an issue, but it's something to consider.

Reply:
moved it under the tapserver